project-base/src/Controller/Front/ErrorController.php

```diff
@@ -46,25 +46,33 @@ class ErrorController extends FrontBaseController
      */
     private $environment;
 
+    /**
+     * @var string|null
+     */
+    protected ?string $overwriteDomainUrl;
```

Comment:
since this is in the `project-base` package, it should be `private` :innocent:

Reply:
👍 85ab620da